cluster/stone-prd-rh01/tenants/rhtap-migration-tenant/consoledot-frontend-standard.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: hac-standard
```

Comment:
I see you changed the name of the file, but not the name of the EnterpriseContractPolicy itself